tests/runner.py

```diff
@@ -134,12 +150,13 @@ def get_browser():
 # Core test runner class, shared between normal tests and benchmarks
 checked_sanity = False
 test_modes = [
-  'default',
+  'asm0',
   'asm1',
   'asm2',
   'asm3',
   'asm2g',
 ]
+default_test_mode = test_modes[0]
```

Comment:
Shouldn't "binaryen0" be the default_test_mode now?

Likewise shouldn't the asm* modes be non-default?

Reply:
Yeah, I wasn't sure about the default mode. But I think you're right, I'll change it to `binaryen0`.

Actually I think we can remove the difference between default and non-default modes - we just need a single default mode for when no mode is specified, but otherwise, that's it. I'll look into that here, I think it'll make things even simpler in the code.